python/nano/src/bigdl/nano/tf/keras/Model.py

```diff
@@ -13,11 +13,98 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+from typing import List, Optional
 
 import tensorflow as tf
+from neural_compressor.model.model import TensorflowBaseModel
+from tensorflow.keras.metrics import Metric
 
 
 class Model(tf.keras.Model):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+    def quantize(self,
```

Comment:
How about `Sequential`. I think you can use the MixedIn technique in Shane's pr #3791

Reply:
Accepted your suggestions in new version. Seems that `Sequential` inherit `Model` according to https://www.tensorflow.org/api_docs/python/tf/keras/Sequential, maybe there is another way to extend original keras layers.

Reply:
I tried that method, seems that `Model` will be broken when inc is not installed for quantization. This is not expected since inc is an option.  
So I bind this function to `tf.keras.Model` as below in `__init__.py`.
```python
try:
    from ..quantization import quantize
    tf.keras.Model.quantize = quantize
except ImportError:
    print("Warning: Intel Neural Compressor should be installed if you need quantization.")
```
And `tf.keras.Sequential` is subclass of `tf.keras.Model`. So we don't need to make extra changes on `Sequential`.  
It occurs to me if we should keep these optional extensions like inc, onnx, ipex in their own folder and test both when they are installed and not installed to make sure everything works fine in real cases.